app/controllers/Clas.scala

```diff
@@ -334,65 +334,6 @@ final class Clas(
       }
     }
 
-  def studentManyForm(id: String) =
```

Comment:
i would prefer that this was all commented out rather than deleted. we may want to bring it back at some point